src/localStorage.js

```diff
@@ -0,0 +1,23 @@
+export const loadState = () => {
+  try {
+    const serializedState = localStorage.getItem('state')
+    
+    if (serializedState === null){
+      return undefined
+    }
+
+    return JSON.parse(serializedState)
+
```

Comment:
If nothing comes back from localStorage, we can just set up initialState as an empty object.